contracts/staking/DelegationManager.sol

```diff
@@ -84,9 +83,14 @@ contract DelegationManager is IDelegationManager, ERC721Full, Lockable {
     Delegator storage delegator = delegators[delegatorId];
 
     require(delegator.lastValidatorEpoch == 0 || delegator.lastValidatorEpoch.add(validatorHopLimit) <= currentEpoch, "Delegation_Limit_Reached");
+
+    address validator;
+    (,,,,,,validator,) = stakeManager.validators(validatorId);
+    require(validator != address(0x0), "Unknown validatorId");
```

Comment:
Unknown validatorId or validator doesn't expect delegations.

Reply:
whatever works, it can be both.

Reply:
actually I am putting both. 😜